### ISSUE TITLE
[bug] Can't paste text via sysytem shortcut key

### ISSUE THREAD

**Issue by zhanglun** [tauri-apps/tauri#3395]
### Describe the bug

I add `<input type="text" />` for inputing. It can input via keyboard, but can't paste text via cmd + v on MacOS.

### Reproduction

1. create a input element
2. try to paste text via sysytem shortcut key, it doest't work

### Expected behavior

paste text into inputs via shortcut key

### Platform and versions

```shell
Operating System - Mac OS, version 12.1.0 X64

Node.js environment
  Node.js - 14.17.0
  @tauri-apps/cli - 1.0.0-beta.10
  @tauri-apps/api - 1.0.0-beta.8

Global packages
  npm - 6.14.13
  yarn - 1.22.15

Rust environment
  rustc - 1.58.0
  cargo - 1.58.0

App directory structure
/node_modules
/public
/src-tauri
/build
/.idea
/src

App
  tauri.rs - 1.0.0-beta.8
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - ../build
  devPath - http://localhost:3000
  framework - React
```

**Comment by amrbashir** [comment]
duplicate of #2397

tldr: on macOS you need to add a menu with native copy+paste+cut, it can be on macOS only if you want using `#[cfg(target_os="macos")]`

**Comment by zhanglun** [comment]
Thanks a lot, I'll give it a try

> duplicate of #2397
> 
> tldr: on macOS you need to add a menu with native copy+paste+cut, it can be on macOS only if you want using `#[cfg(target_os="macos")]`

**Comment by ssbunny** [comment]
Thanks a lot, I'll give it a try too.
